### PR TITLE
issue #8

### PR DESCRIPTION
**Pull request by isaacvv**
Added support for password protected/encrypted PDFs.
PyPDF2 only supports older PDF encryption standards so I used the PyMuPDF (fitz) package for this.

### CODE REVIEW

File: merge_pdf/merge_pdf.py [hunk 1]
```diff
@@ -17,12 +19,30 @@ def pdf_global(pdfs, args, pdf_writer):
...
+        
+        doc = fitz.open("pdf", ibuffer)
+
+        if doc.isEncrypted:
+            if args.password:
+                rc = doc.authenticate(args.password)
+                if not rc > 0:
+                    raise ValueError("wrong password")
+            else:
+                password = input(f"{pdf} is encrypted. Please enter password: ")
```
Comment (kairavkkp) [inline merge_pdf/merge_pdf.py]: Can you try to make the input of the Password Invisible by using getpass() or any Alternative?

**Review by kairavkkp — CHANGES_REQUESTED** [review]
Please see the comments and please add the required. Great Work!
Will merge it after the suggested Changes.

File: merge_pdf/merge_pdf.py (hunk 1 above)
Comment (isaacvv) [reply merge_pdf/merge_pdf.py]: Good idea. I have made the changes

**Review by kairavkkp — COMMENTED** [review]
🎉 LGTM, Great Work!

**Review by kairavkkp — APPROVED** [review]

**Pull request merged**